.github/workflows/vcpkg_ci_amd64.yml

```diff
@@ -175,7 +180,7 @@ jobs:
       - name: 'Rellic build'
         shell: 'bash'
         working-directory: rellic
-        if: contains(matrix.llvm, 'llvm-15') && matrix.target_arch == 'x64'
```

Comment:
Seems that rellic isn't compatible with LLVM 16 https://github.com/lifting-bits/cxx-common/actions/runs/5600543512/jobs/10243080074#step:16:104

Reply:
weird, i thought we updated here https://github.com/lifting-bits/rellic/commit/c40d377d9bcbaeecf08fa0d42e5224dce5a544e5
so i bumped the rellic submodule, maybe i missed something else?

Reply:
Weird... Not sure what the difference is. rellic isn't using the latest cxx-common, so it might be a good idea to test if cxx-common v0.4.1 works (last one with all artifacts available) or not.

Reply:
I opened a Draft PR for rellic here https://github.com/lifting-bits/rellic/pull/329

It seems to build fine (Ubuntu 20.04) but is failing CI due to space issues on the runner.

Reply:
@Ninja3047 if you have time, could you please try changing rellic CI to use Ubuntu 22.04 (you can push to my branch) to see if it fails or not